vnext/Microsoft.ReactNative.ProjectReunion/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
+    <BuildOutDir>$([MSBuild]::NormalizeDirectory('$(SolutionDir)build', '$(Platform)', '$(Configuration)'))\Microsoft.ReactNative.ProjectReunion</BuildOutDir>
```

Comment:
Should we have `<UseWinUI3>true</UseWinUI3>` here?

Reply:
that wouldn't do anything :) That property needs to be set in the Microsoft.ReactNative project (it will control a C preprocessor define that needs to be set so that the resulting microsoft.reactnative.dll is built for winui 3). The actual ProjectReunion dll doesn't do anything with the `UseWinUI3` flag :)

Reply:
sorry yes :)